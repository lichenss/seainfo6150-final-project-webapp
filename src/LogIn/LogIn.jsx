import React from 'react'
import Footer from '../Footer/Footer'
import Header from '../Header/Header'
import styles from './LogIn.module.css'
import { useHistory } from 'react-router-dom'

const LogIn = () => {
    const history = useHistory()
    const handleSubmit=()=>{history.push('/thank-you')}
    return (
        <div>
            <Header />
            <section className={styles.section}>
            <div className={styles.container}>  
            <div className={styles.titleContainer}>
            <h1 className={styles.title}>Log In</h1>
            </div>
            <form className={styles.form} onSubmit={handleSubmit}>
                <div>
                <label className={styles.label}>Name</label>
                <input className={styles.input}
                type="text" required
                />
                </div>
                <div>
                <label className={styles.label}>Email</label>
                <input className={styles.input}
                type="email" required
                />
                </div>
                <div>
                <label className={styles.label}>Content:</label>
                <input className={styles.input}
                type="text" required
                />
                </div>
                <button className={styles.button} type="submit"> Log In</button>
            </form>
            </div>
            </section>
            <Footer />
        </div>
    )
}
export default LogIn