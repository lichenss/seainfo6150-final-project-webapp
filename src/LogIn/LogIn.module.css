.section {
    background-color: rgb(229, 231, 233);
    margin-top: 70px;
}