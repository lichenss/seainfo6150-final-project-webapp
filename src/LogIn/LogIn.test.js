import React from "react";
import LogIn from "./LogIn";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
describe("LogIn tests", () => {
    it("renders correctly", () => {
      const { container } = render(
      <BrowserRouter>
      <LogIn />
      </BrowserRouter>);
      expect(container).toMatchSnapshot();
    });
  });