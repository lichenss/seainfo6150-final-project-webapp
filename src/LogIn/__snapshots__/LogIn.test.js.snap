// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogIn tests renders correctly 1`] = `
<div>
  <div>
    <div>
      <header
        class="header"
      >
        <div
          class="navbar"
        >
          <button
            class="logo"
          />
          <div>
            <a
              class="catLink"
              href="/category/computer-science"
            >
              Computer Science
            </a>
            <a
              class="catLink"
              href="/category/business"
            >
              Business
            </a>
            <a
              class="catLink"
              href="/category/data-science"
            >
              Data Science
            </a>
            <a
              class="catLink"
              href="/category/arts-and-humanities"
            >
              Arts and Humanities
            </a>
            <a
              class="catLink"
              href="/category/physical-science-and-engineering"
            >
              Physical Science and Engineering
            </a>
            <a
              class="catLink"
              href="/category/math-and-logic"
            >
              Math and Logic
            </a>
          </div>
          <div
            class="links"
          >
            <a
              class="barLink"
              href="/log-in"
            >
              Log In
            </a>
            <a
              class="barLink"
              href="/sign-up"
            >
              Sign Up
            </a>
          </div>
        </div>
      </header>
    </div>
    <section
      class="section"
    >
      <div
        class="container"
      >
        <div
          class="titleContainer"
        >
          <h1
            class="title"
          >
            Log In
          </h1>
        </div>
        <form
          class="form"
        >
          <div>
            <label
              class="label"
            >
              Name
            </label>
            <input
              class="input"
              required=""
              type="text"
            />
          </div>
          <div>
            <label
              class="label"
            >
              Email
            </label>
            <input
              class="input"
              required=""
              type="email"
            />
          </div>
          <div>
            <label
              class="label"
            >
              Content:
            </label>
            <input
              class="input"
              required=""
              type="text"
            />
          </div>
          <button
            class="button"
            type="submit"
          >
             Log In
          </button>
        </form>
      </div>
    </section>
    <footer
      class="footer"
    >
      <div
        class="links"
      >
        <a
          class="link"
          href="/contact-us"
        >
          <h3>
            Contact Us
          </h3>
        </a>
        <a
          class="link"
          href="/about-us"
        >
          <h3>
            About Us
          </h3>
        </a>
      </div>
    </footer>
  </div>
</div>
`;
